compiler/rustc_middle/src/mir/syntax.rs

```diff
@@ -409,6 +409,23 @@ impl StatementKind<'_> {
     }
 }
 
+#[derive(
+    Clone,
+    TyEncodable,
+    TyDecodable,
+    Debug,
+    PartialEq,
+    Hash,
+    HashStable,
+    TypeFoldable,
+    TypeVisitable
+)]
+pub enum ExpectKind {
+    True,  // condition is probably true
+    False, // condition is probably false
+    Unpredictable,
```

Comment:
Please make these doc comments, and also explain the last variant.

Reply:
done